site/public/js/server.js

```diff
@@ -955,6 +955,24 @@ function enableTabsInTextArea(id){
 
 }
 
+function changeDisplayOptions(thread_id, option){
+    var url = buildUrl({'component': 'forum', 'page': 'change_display_option'});
+        $.ajax({
+        url: url,
+        type: "POST",
+        data: {
+            thread_id: thread_id,
+            option: option
+        },
+        success: function(data){
+            window.location.replace(buildUrl({'component': 'forum', 'page': 'view_thread', 'thread_id': thread_id, 'option': option}));
+        },
+        error: function(){
+            window.alert("Something went wrong while trying to change views. Please try again.");
+        }
+    });
+}
```

Comment:
What is the point of this ajax call? It doesn't look like 'change_display_option' is a real page in the forum component, nor are you really verifying that said call actually did anything.